apps/passport-client/package.json

```diff
@@ -57,6 +57,8 @@
     "@pcd/artifacts": "0.1.0",
     "@pcd/eslint-config-custom": "*",
     "@types/email-validator": "^1.0.6",
+    "@types/expect": "^24.3.0",
```

Comment:
What is `@types/expect` being used for? I don't see `expect` used anywhere in passport-client